### PR TITLE
Added generator function in advanced level

### PR DESCRIPTION
**Pull request by Vrindagupta6828**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -695,9 +696,42 @@ With capturing, the event is first captured by the outermost element and propaga
...
+  yield i + 1;
+}
+const gen = generator(1);
+console.log(gen.next().value);
+// expected output: 1
+console.log(gen.next().value);
+// expected output: 2 </div>
+```
+* Generators are a special class of functions that simplify the task of writing iterators.
+* A generator is a function that produces a sequence of results instead of a single value, i.e you generate ​a series of values.
```
Comment (annu12340) [inline docs/README.md]: Could you mention these 2 points in the beginning

**Review by annu12340 — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (Vrindagupta6828) [reply docs/README.md]: sure

**Comment by Vrindagupta6828** [comment]
done please merge it now

**Comment by annu12340** [comment]
Please resolve the conflicts

**Comment by annu12340** [comment]
Pls resolve the conflicts

**Comment by annu12340** [comment]
Last and final change 😅
Pls resolve the conflicts

**Comment by Vrindagupta6828** [comment]
@annu12340 hey i am not able to understand what is the conflict. Can you please help?

**Comment by Vrindagupta6828** [comment]
@annu12340 but how to resolve them?

**Comment by Vrindagupta6828** [comment]
@annu12340  please review once again and merge it

**Pull request merged**